.storybook/preview-head.html

```diff
@@ -0,0 +1,8 @@
+<style>
+  #root {
+    display: flex;
```

Comment:
Things are now aligned to the center in storybook

Pet peeve

Reply:
yass

Reply:
yass